src/tls12-download.c

```diff
@@ -210,12 +211,10 @@ int __stdcall entry()
 
     DWORD access_type;
     const wchar_t* proxy_setting;
-    const wchar_t* proxy_bypass_setting;
     if (GetEnvironmentVariableW(L"HTTPS_PROXY", https_proxy_env, sizeof(https_proxy_env) / sizeof(wchar_t)))
     {
         access_type = WINHTTP_ACCESS_TYPE_NAMED_PROXY;
         proxy_setting = https_proxy_env;
-        proxy_bypass_setting = L"<local>";
```

Comment:
I observe tls12-download said `<local>` while the main product said `nullptr`; I think `x-download` is more widely used and thus want to resolve conflicts in favor of it, so I changed this one to nullptr.